src/Strings.php

```diff
@@ -0,0 +1,28 @@
+<?php namespace Lean\Utils;
+
+/**
+ * A suite of functions for working strings.
+ *
+ * Class Strings.
+ *
+ * @package Lean\Utils
+ */
+class Strings
```

Comment:
I think that’s better.


> On Apr 12, 2016, at 6:09 PM, Adam Fenton <notifications@github.com> wrote:
> 
> In src/Strings.php <https://github.com/moxie-leean/wp-utils/pull/1#discussion_r59469881>:
> 
> > @@ -0,0 +1,28 @@
> > +<?php namespace Lean\Utils;
> > +
> > +/**
> > + * A suite of functions for working strings.
> > + *
> > + * Class Strings.
> > + *
> > + * @package Lean\Utils
> > + */
> > +class Strings
> Could change it to Text if you prefer?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/wp-utils/pull/1/files/d04ca263f79a83e43d5173997eb8269004a0c30c#r59469881>